tests/pages/pmmSettingsPage.js

```diff
@@ -232,6 +234,13 @@ module.exports = {
     frequentIntervalValidation: '$frequentInterval-field-error-message',
   },
 
+  switchAzure() {
```

Comment:
It would also be a great help if you could actually implement API function to switchAzure, you already cover using UI for enabling this in one scenario, but you don't need to use the UI every time to make the test faster in other tests you could just enable it via API and perform your next steps.